autopush/senderids.py

```diff
@@ -129,11 +129,11 @@ def senderIDs(self):
         """Return a list of senderIDs"""
         return self._senderIDs
 
-    def get_ID(self, id):
+    def get_ID(self, id=None):
         """Return the associated record for a given SenderID"""
+        if id is None:
```

Comment:
I was originally thinking that we might reserve "" as a default defined in the senderid load table, but that might be overthinking and it's an uncommon use case at best.